ember-bootstrap/addon/components/bs-collapse.ts

```diff
@@ -233,6 +264,10 @@ export default class Collapse extends Component {
    * @protected
    */
   hide() {
+    if (!this._element) {
+      throw new Error('Reference to collapse div element is missing.');
```

Comment:
Should we use `assert` from `@ember/debug` here? This would strip the code in a production build, but still make TS happy.

Reply:
Done in a8a06c4a2b61380768df14feeecc0219cf079a4d. Wasn't aware that `assert` from `@ember/debug` is able to narrow the type.